scripts/run-wsdunit

```diff
@@ -35,7 +35,7 @@ do
     echo ">>> $(date +%b-%d) @ $(date +%H:%M:%S) Run #$i (of $limit):";
     ./coolwsd --disable-cool-user-checking --cleanup &> /dev/null
     (cd test && ./run_unit.sh --test-name $name.la --log-file $name.log --trs-file $name.trs --color-tests yes --enable-hard-errors yes --expect-failure no -- ./$name.la)
-    if test $? -eq 0; then
+    if test $? -eq 0 && grep -q PASS test/$name.trs; then
```

Comment:
Unfortunately, grepping is still needed, in case we don't fail-fast and `exit 1` from run_unit.sh.